packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -0,0 +1,547 @@
+import {ProcessWarningIcon} from '@twilio-paste/icons/esm/ProcessWarningIcon';
+
+import {
+  ProcessDisabled,
+  ProcessDraft,
+  ProcessError,
+  ProcessInProgress,
+  ProcessNeutral,
+  ProcessSuccess,
+  ProcessWarning,
+  ConnectivityAvailable,
+  ConnectivityBusy,
+  ConnectivityUnavailable,
+  ConnectivityOffline,
+  processStatusExamples,
+  connectivityStatusExamples,
+  prefixSuffixTooltipExamples,
+  tableExample,
+  tableWithPlainTextExample,
+  linkInStatusExamples,
+  otherCommonPlacements,
+  BannerExample,
+} from '../../../component-examples/StatusPatternExamples.tsx';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/patterns/status/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Status"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+            Product_suitability
+          }
+        }
+      }
+    }
+  }
+`;
+
+<content>
+
+<PatternHeader
+  description={props.pageContext.frontmatter.description}
+  name={props.pageContext.frontmatter.title}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{Box, Text, ProcessSuccessIcon}}>{BannerExample}</LivePreview>
+
+## Ingredients
+
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/icons">Icon</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/primitives/text">Text</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="/primitives/box">Box</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import all components for Status patterns
+
+import {​ Text } from "@twilio-paste/core/text";
+import { Box } from '@twilio-paste/core/box';
+import { ProcessDisabledIcon } from '@twilio-paste/icons/esm/ProcessDisabledIcon';
+import { ProcessDraftIcon } from '@twilio-paste/icons/esm/ProcessDraftIcon';
+import { ProcessErrorIcon } from '@twilio-paste/icons/esm/ProcessErrorIcon';
+import { ProcessInProgressIcon } from '@twilio-paste/icons/esm/ProcessInProgressIcon';
+import { ProcessNeutralIcon } from '@twilio-paste/icons/esm/ProcessNeutralIcon';
+import { ProcessSuccessIcon } from '@twilio-paste/icons/esm/ProcessSuccessIcon';
+import { ProcessWarningIcon } from '@twilio-paste/icons/esm/ProcessWarningIcon';
+
+```
+
+## Usage
```

Comment:
Why show a "Usage" section if we don't have any content for it?

Reply:
I think "General Usage" and "Accessibility" are both supposed to be under "Usage". I just added a suggestion to adjust "General Usage" to an H3.